src/Lucid/Model/index.js

```diff
@@ -483,28 +483,11 @@ class Model extends BaseModel {
    * @private
    */
   _instantiate () {
-    this.__setters__ = [
-      '$attributes',
-      '$persisted',
-      'primaryKeyValue',
-      '$originalAttributes',
-      '$relations',
-      '$sideLoaded',
-      '$parent',
-      '$frozen',
-      '$visible',
-      '$hidden'
-    ]
-
-    this.$attributes = {}
-    this.$persisted = false
-    this.$originalAttributes = {}
-    this.$relations = {}
-    this.$sideLoaded = {}
-    this.$parent = null
-    this.$frozen = false
-    this.$visible = null
-    this.$hidden = null
+    super._instantiate()
```

Comment:
Good catch. I am not sure why I even had to `instantiate` methods at first place. Also I believe it's find to have `$visible` and `$hidden` properties on the BaseModel itself and simply remove `_instantiate`  from here.